### PR TITLE
Update version to 4.1.8

### PR DESCRIPTION
**Pull request by jsvob**
see the changelog

### CODE REVIEW

**Review by costaconrado — APPROVED** [review]
LGTM.

**Review by osoukup — APPROVED** [review]
LGTM

File: docs/CHANGELOG.md
Comment (jobselko) [inline docs/CHANGELOG.md:1]: Should not this be a minor release? There was an API change.

**Review by JakubFrejlach — CHANGES_REQUESTED** [review]
Agree with @jobselko, this should be minor release 4.2.0

**Pull request closed without merging**